addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerJsonParser.java

```diff
@@ -66,12 +72,33 @@ public WappalyzerJsonParser() {
 
     WappalyzerData parse(String categories, List<String> technologies, boolean createIcons) {
         LOGGER.info("Starting to parse Tech Detecction technologies.");
+        Instant start = Instant.now();
         WappalyzerData wappalyzerData = new WappalyzerData();
         parseCategories(wappalyzerData, getStringResource(categories));
-        technologies.forEach(
-                path -> parseJson(wappalyzerData, getStringResource(path), createIcons));
+        // Process the files/paths in parallel
+        ExecutorService executor =
+                Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() - 1);
```

Comment:
- Worth checking with `commonlib`'s `Constants`.
 - Better shutdown the executor once no longer needed.
 - Name the threads, if there's a problem it's easier to identify the what.